src/cris-test-harness-cli/package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "cris-test-harness-cli",
+  "version": "0.3.0",
+  "description": "Package for running the cris-test-harness from the command line",
+  "main": "lib/main.js",
+  "types": "lib/main.d.ts",
+  "scripts": {
+    "build": "npx tsc"
+  },
+  "author": "@jolly59, @nhuntwalker",
```

Comment:
@KatieProchilo , should be added as well.

Reply:
Done